Gopkg.toml

```diff
@@ -40,6 +40,10 @@
   name = "github.com/gorilla/websocket"
   version = "=1.2.0"
 
+[[constraint]]
+  name = "github.com/rs/cors"
+  version = "=1.6.0"
```

Comment:
I think we can lessen the version requirement here. 

@ebuchman What is the reason for tendermint requiring precise versions again? This results in great pain when integrating TM (see https://github.com/tendermint/tendermint/issues/2798). Versions are already frozen in `.lock` file. Plus it's possible to update one dependency without updating the other one.

Reply:
I think so too. Only one reason I did precise version - all versions in toml are precise. I will change to **^1.6.0**. Also, do you plan to use go modules?

Reply:
It's to prevent dependencies from being upgraded uncontrollably/accidentally.

The "great pain" should be overcomeable by using [override], no?

Reply:
Finished with plain **1.6.0**. I think, it's better to revise dependencies versions in separate issue.

Reply:
> uncontrollably/accidentally

I think this can easily be prevented by a) using `dep ensure -update XXX` to update single dependency b) review process.

> The "great pain" should be overcomeable by using [override], no?

I want for our users to have **no pain by default** We shouldn't put our problems on their shoulders.

Reply:
I somehow didn't know about `dep ensure -update XXX`. Is that new? 

What do you recommend we change this too instead of `=` ?

Reply:
> I somehow didn't know about dep ensure -update XXX. Is that new?

no. it existed as long as I remember dep

> What do you recommend we change this too instead of = ?

Depending on a library. I think sticking with `^` major version is fine for most of the libs. Some require locking onto minor version '~`